modules/Core/pages/mod/punishments.php

```diff
@@ -189,6 +192,25 @@
             }
 
             try {
+                $infractionTypeText = "";
+                if($infraction->type == 1){
+                    $infractionTypeText = $language->get('mod', 'ban');
+                }else if($infraction->type == 2){
+                    $infractionTypeText = $language->get('mod', 'warn');
+                }else if($infraction->type == 3){
+                    $infractionTypeText = $language->get('mod', 'ban_ip');
+                }
+
+                $queries->create('logs', array(
+                        'time' => date('U'),
+                        'action' => $language->get('logs', 'log_puishment_revoke'),
+                        'user_id' => $user->data()->id,
+                        'ip' => $user->getIP(),
+                        'info' => $language->get('mod', 'punishment')."{$infractionTypeText}: ".$language->get('user', 'username').": ".Output::getClean(
+                            $queries->getWhere("users", ["id", "=", Output::getClean($_GET['user'])])[0]->username
```

Comment:
Unfortunately,
```
Output::getClean($queries->getWhere("users", ["id", "=", Output::getClean($_GET['user'])])[0]->username)
```
is not compatible with PHP < 5.5 if I remember correctly, and Nameless is for 5.4+.

Can this be changed to:
```
$query_user = $queries->getWhere("users", ["id", "=", Output::getClean($_GET['user'])]);
$query_user = Output::getClean($query_user[0]->username);

$queries->create('logs', array(
  ......,
  'info' => $language->get('mod', 'punishment')."{$infractionTypeText}: ". $query_user,
```